Makefile

```diff
@@ -1,9 +1,18 @@
+SETUP_FILE ?= ./setup.mk
+
+# The setup file may define RUNENV for runtime settings, e.g.
+# RUNENV ?= DYLD_FALLBACK_LIBRARY_PATH=${DEVENVPREFIX}/lib
+ifneq (,$(wildcard $(SETUP_FILE)))
+	include $(SETUP_FILE)
+endif
+
 SHELL = /bin/bash
-PYTHON := $(shell which python3 2>/dev/null)
+PYTHON_EXE := $(shell which python3 2>/dev/null)
+PYTHON := env ${RUNENV} ${PYTHON_EXE}
```

Comment:
This trick raise a regression  of `make install`. See https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_node/make_79.html `If the file name contains no slash, the directory part is the string `./'.`. That being said, the `dir` evaluated value of `PREFIX ?= $(realpath $(dir ${PYTHON})/../)` is `./ ./some-real-path`. This makes `PREFIX` in `.PHONY install` session is unexpected, and `cp libmarch.*.so ${PYTHON_LIBRARY_DIR}` will fail.

Reply:
I am not sure I can interpret the literals  correctly because the formatting seems garbled.

Does #240 address this issue?

Reply:
Yeeeeeeeeeeeeeeeeeeeeeeeeeees!!!!!!

```
...................................................................................................................................................................................................................................
................................................................................................................................................................SSS                                                                
----------------------------------------------------------------------                                                                                                                                                             
Ran 390 tests in 3.998s                                                                                                                                                                                                            
                                                                                                                                                                                                                                   
OK (SKIP=3)                                                                                                                                                                                                                        
.                                                                                                                                                                                                                                  
----------------------------------------------------------------------                                                                                                                                                             
Ran 1 test in 0.282s                                                                                                                                                                                                               
                                                                                                                                                                                                                                   
OK 
```